src/Account/TSAttributes.m

```diff
@@ -13,18 +13,21 @@
 
 @implementation TSAttributes
 
-+ (NSDictionary *)attributesFromStorageWithVoiceSupport {
++ (NSDictionary *)attributesFromStorage:(BOOL)isWebRTCEnabled {
     return [self attributesWithSignalingKey:[TSStorageManager signalingKey]
-                            serverAuthToken:[TSStorageManager serverAuthToken]];
+                            serverAuthToken:[TSStorageManager serverAuthToken]
+                            isWebRTCEnabled:isWebRTCEnabled];
 }
 
 + (NSDictionary *)attributesWithSignalingKey:(NSString *)signalingKey
                              serverAuthToken:(NSString *)authToken
+                             isWebRTCEnabled:(BOOL)isWebRTCEnabled
 {
     return @{
         @"signalingKey" : signalingKey,
         @"AuthKey" : authToken,
-        @"voice" : [NSNumber numberWithBool:YES], // all Signal-iOS clients support voice
+        @"voice" : @(YES), // all Signal-iOS clients support voice
```

Comment:
Not being sure how a boolean object get's serialized into the ultimate HTTP request (e.g. `false` or `0` ?), I would be cautious here. 

Signal-iOS doesn't use this value when pulled from the contact intersection request anyway, but Signal-Android currently does (though apparently that's going away with the webrtc branch). So the failure case would be that these Signal-iOS users wouldn't be callable from Signal-Android.

Can you confirm it's the same?